command_helper/get_documentation.py

```diff
@@ -0,0 +1,86 @@
+from os import path
+try:

```

Comment:
@aaltat as far as I understood you are dropping support of Sublime Text 2 (and that is cool).
So, there is no need to keep imports for both versions here and in all other modules where the same imports approach is used (try to import modules for sublime2 -> except -> import for sublime3).